### ISSUE TITLE
what does this v2.0.3 warning mean in the docs?

### ISSUE THREAD

**Issue by gregveres** [victorgarciaesgi/vue-chart-3#73]
## Describe the bug
In the docs, there is a warning:
`⚠️ Since 2.0.3 (for Vue 2 users), Vue composition api will not register itself. Update your project if you didn't do it manually!
`
What does that mean? In my main, I already register the composition api for my app:

```
import Vue from 'vue';

import '@plugins/composition-api';
import { vuetify } from '@plugins/vuetify';
import { i18n } from '@plugins/i18n';
import '@plugins/axios';
import '@plugins/chartjs';

import App from '@pages/app/App.vue';

import { SetBrowserTitle } from '@helpers/WindowHelpers';
import { router } from './router';

Vue.config.productionTip = false;

new Vue({
  router,
  vuetify,
  i18n,
  render: (h) => h(App)
}).$mount('#app');
```
@plugins/composition-api.ts is:
```
import Vue from 'vue';
import VueCompositionApi from '@vue/composition-api';

Vue.use(VueCompositionApi);
```
@plugins/chartjs.ts is:
```
import { Chart, LineController, BarController, Tooltip, Title, Legend } from 'chart.js';

Chart.register(LineController, BarController, Tooltip, Title, Legend);
```

But when I try to visit a page that has a chart on it, I am getting this error:
```
vue.runtime.esm.js:619 [Vue warn]: Error in data(): "Error: [vue-composition-api] must call Vue.use(VueCompositionAPI) before using any function."

found in

---> <LineChart>
       <LoadingIndicator> at src/components/LoadingIndicator/LoadingIndicator.vue
         <SessionLeagueStats> at src/components/LeagueComponents/SessionLeagueStats/SessionLeagueStats.vue
```
<LineChart> is your line chart class. 
I am using v2.0.3 with Vue 2.

I must have missed something but for the life of me, I can't figure out what. Any suggestions? Or some clearification of what the warning means in the docs?

BTW, thank you very much for this project. I am trying to migrate from vue-chartjs. 

## Version of `vue-chart-3`
v2.0.3

## Version of Vue

- [ ] Vue 3
- [x] Vue 2
- [ ] Nuxt 2
- [ ] Nuxt 3

**Comment by gregveres** [comment]
Looking at the errors that follow the error I copied above, I see the very next error is this:
```
vue.runtime.esm.js:619 [Vue warn]: `createElement()` has been called outside of render function.
```
This happens after my axios request returns and we now have data to render in the chart.

**Comment by gregveres** [comment]
This issue can be closed. It turned out that upgrading the vue-compsosition-api plugin fixed the issue. I was on v1.3.3 and I upgraded to the latest available at this time: v1.4.3.
